### PR TITLE
reduce duplicative check test setup

### PR DESCRIPTION
**Pull request by rocktavious**
## Issues

Part 1 of the cleanup

### CODE REVIEW

File: check_test.go [hunk 1]
```diff
@@ -1,609 +1,694 @@
...
+	fixture autopilot.TestRequest
+	body    func(c *ol.Client) (*ol.Check, error)
 }
 
-var checkUpdateNotes = "Hello World Check"
+// Helper Data
+var (
+	templater = template.New("").Funcs(sprig.TxtFuncMap()).Delims("[%", "%]")
+
+	id = ol.ID("Z2lkOi8vb3BzbGV2ZWwvMTIzNDU2")
```
Comment (davidbloss) [inline check_test.go:26]: Just an FYI there are ids for tests in [clientGQL_test.go](https://github.com/OpsLevel/opslevel-go/blob/main/clientGQL_test.go#L25)

**Review by davidbloss — APPROVED** [review]
LGTM1 🎉

**Comment by rocktavious** [comment]
Somehow all of this got added to main from a different way.

**Comment by rocktavious** [comment]
It appears to have come in through the https://github.com/OpsLevel/opslevel-go/pull/442 as a dependent commit

**Pull request closed without merging**